core/domain/config_domain.py

```diff
@@ -290,7 +300,7 @@ def set_value(self, committer_id, raw_value):
                 model_instance.id: model_instance.value
             })
 
-    def normalize(self, value):
+    def normalize(self, value: str) -> Any:
```

Comment:
Explain why this returns `Any`

Reply:
In this function we are calling `normalize_against_schema` from `schema_utils.py`.
https://github.com/oppia/oppia/blob/6431446ad481100a8d71d79ab5636c0e6c72aef6/core/schema_utils.py#L78
so, `normalize_against_schema` returning object of `Any` type which was ultimately returned by this function.

Reply:
add a comment into the code

Reply:
done!